docs/pages/components/dialog.md

```diff
@@ -269,6 +280,15 @@ const App = () => {
 };
 ```
 
+:::tip
+**Match a dialog’s variant to specific buttons and icons:**
+| Dialog variant | Button variant | Icon name |
+| ----------------------- | -------------- | ---------------------- |
+| `warning` | `warning` | `exclamation-triangle` |
+| `default` confirmation | `primary` | `exclamation-circle` |
+| `default` informational | `primary` | `info-circle` |
```

Comment:
Should the `default`s still be in there?

Reply:
Yes, they are two separate types of dialogs that should use the `default` variant. But your comment made me realize the table could be laid out more clearly, so I made an update! 

<img width="843" alt="Screenshot 2024-07-17 at 15 22 28" src="https://github.com/user-attachments/assets/8d2cfb6c-9f26-4f11-87ca-dac45f80344f">